apps/CommonApp/src/new_api/camera/AnimatedCameraView.tsx

```diff
@@ -1,55 +1,56 @@
 import React from 'react';
-import { CameraView, useCameraPermissions } from 'expo-camera';
+import { CameraView, useCameraPermissions, CameraViewProps } from 'expo-camera';
 import { Button, StyleSheet, Text, View } from 'react-native';
 import Animated, {
-  SharedValue,
+  AnimateProps,
   useAnimatedProps,
 } from 'react-native-reanimated';
 
 const AnimatedCameraView = Animated.createAnimatedComponent(CameraView);
 
-interface CameraProps {
-  facing: 'front' | 'back';
-  zoom: SharedValue<number>;
+interface CameraProps extends Omit<CameraViewProps, 'zoom'> {
+  ref?: React.RefObject<
+    React.Component<AnimateProps<CameraViewProps>, any, any>
+  >;
+  zoom: Animated.SharedValue<number>;
 }
 
-const Camera = React.forwardRef(
-  (props: CameraProps, ref: React.Ref<CameraView>) => {
-    const [permission, requestPermission] = useCameraPermissions();
+const Camera = ({ ref, ...props }: CameraProps) => {
+  const [permission, requestPermission] = useCameraPermissions();
 
-    const animatedProps = useAnimatedProps(() => {
-      return {
-        zoom: props.zoom.value - 1,
-      };
-    });
+  const animatedProps = useAnimatedProps(() => {
+    return {
+      // @ts-ignore zoom value is correct
```

Comment:
What was the error here?

Reply:
Seems like this one was also turned into a warning (I believe it was `@typescript-eslint/no-unsafe-member-access`, but I could be wrong)